src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -191,10 +190,32 @@ static boolean hasOcr(CaseDetails theCase) {
             .map(Try::toValidation)
             .map(validation -> validation
                 .mapError(throwable -> "Invalid journeyClassification. Error: " + throwable.getMessage())
+                .flatMap(classification -> validateClassification(classification, theCase))
             )
             .orElse(invalid("Missing journeyClassification"));
     }
 
+    private static Validation<String, Classification> validateClassification(
+        Classification classification,
+        CaseDetails theCase
+    ) {
+        if (SUPPLEMENTARY_EVIDENCE.equals(classification)) {
+            return invalid(format(
+                "Event createCase not allowed for the current journey classification %s",
```

Comment:
Having had some thoughts about it, I think we should get rid of the whole "Event createCase" part, as this code doesn't really know what event is being handled. Also, this message would be displayed to the caseworker, who doesn't understand this value. It's Ok to just say sth like "Classification X doesn't allow for creating new cases" or sth like that.

Reply:
That's not what ticket suggests. Shall we continue with our BA before making a change?

Reply:
Oh, you're right. Yes, let's have a quick chat.

Reply:
Ok, this can always be refined.